src/Microsoft.AspNet.SignalR.SqlServer/TaskAsyncHelper.cs

```diff
@@ -0,0 +1,1194 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
We want to get rid of it completely though. Starting with the scaleouts since they are easier and currently being worked on. There are 3 locations it is being used right now in this project. It would be great if we could remove them.